docs/language/composite-types.mdx

```diff
@@ -79,7 +79,7 @@ of the composite type requires them.
 
 Composite types can only be declared within [contracts](../contracts)
 and not locally in functions.
-They can also not be nested.
```

Comment:
Was this line removed on purpose? If it doesn't make sense, I think we need to extend it / improve the wording

Reply:
I removed this line on purpose indeed. When you scroll up a little bit on that page (composite-types), there is written: "Nesting of resources is only allowed within other resource types, or in data structures like arrays and dictionaries, but not in structures, as that would allow resources to be copied." Since resources are composite types, this means that composite types can be nested in some cases. That's why I removed that line. Does it make sense? Or maybe am I misunderstanding the writer of that sentence?

Reply:
Yeah, I guess we can remove that sentence 👍